docs/references/javascript/user/totp.mdx

```diff
@@ -95,3 +103,269 @@ Generates a fresh new set of backup codes for the user. Note that if this method
 | `codes` | `string[]` | The generated set of backup codes |
 | `createdAt` | `Date` | Creation date of the TOTP secret |
 | `updatedAt` | `Date` | Update timestamp of the TOTP secret |
+
+## TOTP example
+
+The following example demonstrates how to use the TOTP methods to create and verify a TOTP secret, disable TOTP, and create backup codes for a user. Upon any error, the example will display the error message to the user.
```

Comment:
```suggestion
The following example demonstrates how to use the TOTP methods to create and verify a TOTP secret, disable TOTP, and create backup codes for a user. Upon any error, the example will display the error message to the user.

<Callout type="Info">
This example requires you to enable **Authenticator application** and **Backup codes** as MFA strategies in your Clerk dashboard.
</Callout>
```

Didn't realize I had to do this

Reply:
Yes, it's in a callout at the top. I can make the callout at the top more clear, or I can add the callout here as well? We need the callout at the top for sure, or we could add the callout to each method, but its for sure the first step users should do before trying to use any of the methods.

Reply:
Ah, I see. Yeah, I totally missed it at the top, but I think I would've read it if it was above the intro paragraph, so maybe that's the solution.

Is an authenticator app required for using TOTP? If so, we should mention it specifically in the callout